tests/js/setup.js

```diff
@@ -159,13 +158,17 @@ window.TestStubs = {
         'team:write',
         'team:admin',
       ],
+      status: {
+        id: 'active',
+        name: 'active',
+      },
       features: [],
       onboardingTasks: [],
       teams: [],
       ...params,
     };
   },
-  Repository: (...params) => {
+  Repository: params => {
```

Comment:
when i implemented this in Zeus I noticed the incorrect use of ...params expansion here (which wont convert to dict values later)